{{cookiecutter.extension_name}}/RELEASE.md

```diff
@@ -0,0 +1,30 @@
+# Making a {{ cookiecutter.extension_name }} release
+
+This document guides a contributor through creating a release of {{ cookiecutter.extension_name }}.
+
+## Check installed tools
+
+### Clean the repository
+
+To ensure a clean build, you can remove all non-tracked files with:
+
+```bash
+git clean -xfdi
+```
+
+This would ask you for confirmation before removing all untracked files.
+
+### Create the release
+
+We publish a Python source package and a Python universal binary wheel.
+See the Python docs on [package uploading](https://packaging.python.org/distributing/#uploading-your-project-to-pypi)
+for twine setup instructions and for why twine is the recommended uploading method.
```

Comment:
Link [twine](https://github.com/pypa/twine) 

Use GitHub since there are no docs for twine